### PR TITLE
docs: improved the zh and en version of sql/task_manage/show_jobs

### PR DESCRIPTION
**Pull request by michelle-qinqin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
fixed the sql example 
improved the format

### CODE REVIEW

File: docs/en/reference/sql/task_manage/SHOW_JOBS.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
 # SHOW JOBS
 
+The `SHOW JOBS` statement displays the list of tasks that have been submitted in online mode.
```
Comment (dl239) [inline docs/en/reference/sql/task_manage/SHOW_JOBS.md]: in standalone or offline mode in cluster

Comment (michelle-qinqin) [reply docs/en/reference/sql/task_manage/SHOW_JOBS.md]: 是只有单机版和集群版的离线模式支持的意思吗？
但是我实践的时候发现单机版似乎不支持？
然后集群版的在线和离线模式都是支持的

File: docs/zh/reference/sql/task_manage/SHOW_JOBS.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 # SHOW JOBS
 
+`SHOW JOBS`语句显示在线模式下已经提交的任务列表。
```
Comment (lumianph) [inline docs/zh/reference/sql/task_manage/SHOW_JOBS.md]: 集群版的离线和在线模式

File: docs/en/reference/sql/task_manage/SHOW_JOBS.md (hunk 1 above)
Comment (lumianph) [reply docs/en/reference/sql/task_manage/SHOW_JOBS.md]: in offline or online mode of the cluster version

**Review by dl239 — APPROVED** [review]

File: docs/en/reference/sql/task_manage/SHOW_JOBS.md [hunk 3]
```diff
@@ -1,16 +1,18 @@
 # SHOW JOBS
 
+The `SHOW JOBS` statement displays the list of tasks that have been submitted in offline or online mode of the cluster version.
```
Comment (vagetablechicken) [inline docs/en/reference/sql/task_manage/SHOW_JOBS.md]: online mode only `LOAD DATA`, others won't be shown in here.
is it better?
```
submit to taskmanager, all offline job, online `LOAD DATA` job
```

Comment (michelle-qinqin) [reply docs/en/reference/sql/task_manage/SHOW_JOBS.md]: Is it ok if I change it to :

The `SHOW JOBS` statement displays the list of tasks that have been submitted to taskmanager in the cluster version, including all kinds of jobs in offline mode and `LOAD DATA` jobs in online mode

**Review by lumianph — APPROVED** [review]

**Pull request merged**